README.md

```diff
@@ -57,24 +50,47 @@ With simple commands, this bot allows you to:
 
 [![Demo Video](https://i9.ytimg.com/vi/aCjcT1CHAzU/mq3.jpg?sqp=COSotI0G&rs=AOn4CLD34jFIlq6GRdmTnK6p3F8O2F-Yig)](https://youtu.be/aCjcT1CHAzU)
 
-## What is new in this version?
+## What's new?
+
+<#TODO: short description here>
 
 Checkout the [this documentation](https://github.com/usmanwardag/dollar_bot/blob/main/docs/Updated_version.pdf)
+
 ## Installation
 
-The below instructions can be followed in order to set-up this bot at your end in a span of few minutes! Let's get started:
+#### Pre-requisite: The Telegram Desktop App
```

Comment:
Header size looks smaller than others. Any specific reason for that?

Reply:
Yeah, since it's an external package install step that's just a small pre-req before the actual DollarBot installations, I think it's warranted.